pyproject.toml

```diff
@@ -237,6 +235,7 @@ dev = [
     "mkdocs-awesome-pages-plugin",
     "mkdocstrings",
     "mike",
+    "maison",
```

Comment:
What is this?

Reply:
I was planning to dig a bit deeper into this. As I was playing around with the dependencies yesterday, I removed some unused requirements and I think, the relaxed requirements led to a version of `maison` greater than 2.0, which ultimately forces `yamlfix` to fail.

I also saw, as of last week, that we apply a similar fix when we use `install-zenml-dev` script. Perhaps @avishniakov can shed more light on this as he must have bumped into a similar error [here](https://github.com/zenml-io/zenml/blob/bda5e888ea6e323d25a2c72deba19c81e4c7201c/scripts/install-zenml-dev.sh#L63).

Reply:
I am resolving this then.